docs/multi-round.md

```diff
@@ -112,7 +112,12 @@ Processor instance will only be created once, which means you can store informat
 
 All KSP symbols will not be reusable cross rounds, due to the resolution result can potentially change based on what was generated last round. However, since KSP does not allow modifying existing code, information such as the string value for a name of a symbol should still be reusable. To summarize, processors can keep information from previous round on their own, but need to bear in mind that they might be invalid in future rounds.
 
+#### Error Handling
+In case of an error (defined by processor calling `KSPLogger.error()`), the round where error occurs will be the last round of processing. All processors will invoke `onError()` method, there will be no `finish()` method invoked in case of an error.
```

Comment:
"`onError()` will be invoked for all processors" might be better; Processors don't invoke `onError()` actively.